examples/helloworld/src/index.js

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import _ from 'underscore';
 import App from './App';
 
-React.render(<App />, document.getElementById('root'))
+
+var state = {
+    very: {
+        deeply: {
+            nested: {
+                counts: _.range(400).map(function () { return 0; })
+            }
+        }
+    }
+};
+
+function swapper(f) {
+    state = f(state);
+    queueRender();
+}
+
+function queueRender() {
+    var cur = ReactCursor.Cursor.build(state, swapper);
+    React.render(<App cursor={cur} />, document.getElementById('root'));
```

Comment:
I don't understand, why does this matter? The root cursor was rebuilt each render in the old pattern too (inside RootComponent.render)